gui/components/community/authenticate_popup.py

```diff
@@ -24,4 +24,4 @@ def wait_until_appears(self, timeout_msec: int = configs.timeouts.UI_LOAD_TIMEOU
     def authenticate(self, password: str):
         self._passwort_text_edit.type_text(password)
         self._authenticate_button.click()
-        self.wait_until_hidden()
+        self._authenticate_button.wait_until_hidden()
```

Comment:
Changed it here because otherwise my test thinks that popup is not hidden (because after that another popup appears)